packages/core/src/browser/style/alert-messages.css

```diff
@@ -23,24 +23,34 @@
 .theia-alert-message-container i {
     padding-right: 3px;
 }
- 
+
+/* success message */
+.theia-success-alert {
+    border: var(--theia-border-width) solid var(--theia-successBackground);
```

Comment:
background colors for borders?

Reply:
I'm trying to find the best approach for a `success` color at the moment.
In `core` we define:

https://github.com/eclipse-theia/theia/blob/f015148a680e3c5ca892a6050b7cd8d86e03f413/packages/core/src/browser/common-frontend-contribution.ts#L1597-L1603

Which only gets a default value from `scm`:

https://github.com/eclipse-theia/theia/blob/f015148a680e3c5ca892a6050b7cd8d86e03f413/packages/scm/src/browser/scm-contribution.ts#L199-L204

It is however used in areas outside of `scm` (and is also used a border-color):

https://github.com/eclipse-theia/theia/blob/f015148a680e3c5ca892a6050b7cd8d86e03f413/packages/scm/src/browser/style/index.css#L124-L126

Reply:
How can I get some element with success styles? I wonder whether we really need it. VS Code does not seem to have it.

Reply:
> How can I get some element with success styles? I wonder whether we really need it. VS Code does not seem to have it.

Correct, they do not have success styling which is why we had to create our own.
I think we should verify if such styling is truly required.

Reply:
i think the PR look ok besides this point. Do you think we should tackle this issue separately? Could you file an issue?

Reply:
> i think the PR look ok besides this point. Do you think we should tackle this issue separately? Could you file an issue?

Yes I think so, I'll file an issue so we could tackle it separately.